gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.2-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.3-all.zip
```

Comment:
@bhaktideshmukh This isn't necessary for this PR.

Reply:
@bhaktideshmukh this line is still here.

Reply:
@gouri-panda I removed that gradle file and sorry for the delay. I didn't realized that file wasn't removed.